### PR TITLE
Rename module to match github repo

### PR DESCRIPTION
**Pull request by vearutop**
# Description

Recently this repo was extracted from `common`, this PR changes module name so that it matches new repo and can be discovered by `go get`.

# Motivation & context

Fixes `go get`

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Note to other contributors

No note.

## Update required of cucumber.io/docs

No update.

# Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](../CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by aslakhellesoy** [comment]
Thanks @vearutop - could you please add an entry to `CHANGELOG.md`?

**Review by bkielbasa — APPROVED** [review]

File: go/go.sum [hunk 1]
```diff
@@ -3,7 +3,6 @@ github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
-github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
```
Comment (mattwynne) [inline go/go.sum:6]: What does this bit do? Is this file just auto-generated by something?

**Review by mattwynne — APPROVED** [review]

File: go/go.sum (hunk 1 above)
Comment (aurelien-reeves) [reply go/go.sum:6]: Yes, this is autogenerated by go

**Comment by vearutop** [comment]
I've added a note to changelog, please check.

File: go/go.sum (hunk 1 above)
Comment (vearutop) [reply go/go.sum:6]: Yes, I did `go mod tidy` to cleanup/actualize dependencies graph.

**Review by aurelien-reeves — APPROVED** [review]
Looks good to me 👌

**Comment by aslakhellesoy** [comment]
@mattwynne @aurelien-reeves the current settings doesn't allow me to merge this PR, because the checks haven't completed.

Do you know how to make sure the checks (GitHub actions) actually run? The PR is from a fork - could that be the reason why they are currently not running?

**Comment by aslakhellesoy** [comment]
The action declares this trigger event for pull requests:

https://github.com/cucumber/cucumber-expressions/blob/0938a9a699beb279469fe18505af25fe649aaa24/.github/workflows/test-go.yml#L6-L7

Is that a typo? From what I can tell, a `pull_request` event [doesn't have a](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request) `merged` type. Could this be why it's not run? Do we even need a `types` filter?

**Comment by vearutop** [comment]
Yes, I think this is because of first-time contrib, I don't have a button to trigger ci, and also no merge button.

**Comment by aslakhellesoy** [comment]
@aurelien-reeves I’ve merged #26 and updated this PR, but still bot able to trigger CI for this PR. Any ideas?

**Comment by aslakhellesoy** [comment]
Hum - seems fine now

**Comment by aurelien-reeves** [comment]
Did you do something to make it run the CI?

**Pull request merged**